lib/extensions/postgres_cdc_rls/migrations.ex

```diff
@@ -118,26 +118,32 @@ defmodule Extensions.PostgresCdcRls.Migrations do
         db_password
       )
 
-    Repo.with_dynamic_repo(
-      [
-        hostname: host,
-        port: port,
-        database: name,
-        password: pass,
-        username: user,
-        pool_size: 2,
-        socket_options: db_socket_opts
-      ],
-      fn repo ->
-        Ecto.Migrator.run(
-          Repo,
-          @migrations,
-          :up,
-          all: true,
-          prefix: "realtime",
-          dynamic_repo: repo
-        )
-      end
-    )
+    db_config = [
+      hostname: host,
+      port: port,
+      database: name,
+      password: pass,
+      username: user,
+      pool_size: 2,
+      socket_options: db_socket_opts
+    ]
+
+    ssl_enforced = Map.get(args, "ssl_enforced", true)
```

Comment:
If we wrap this into a function, we can consolidate the default value in one place. Currently, we declare the default behavior in three different places

```elixir
  @spec get_default_ssl_param(map()) :: boolean()
  def get_default_ssl_param(args) do
    !!Map.get(args, "ssl_enforced", true)
  end
```